Cargo.toml

```diff
@@ -20,7 +21,7 @@ default = ["termion"]
 [dependencies]
 bitflags = "1.0"
 cassowary = "0.3"
-itertools = "0.7"
+itertools = "0.8"
```

Comment:
Is there a reason for this update to be part of this PR ? I try to update all dependencies before each major release and this require some testing to be sure that no strange behaviors are introduced.